Makefile

```diff
@@ -3,8 +3,9 @@ PROJECT_NAME := Pulumi Amazon Web Services (AWS) EKS Components
 # Override during CI using `make [TARGET] PROVIDER_VERSION=""` or by setting a PROVIDER_VERSION environment variable
 # Local & branch builds will just used this fixed default version unless specified
 PROVIDER_VERSION ?= 3.0.0-alpha.0+dev
-# Use this normalised version everywhere rather than the raw input to ensure consistency.
-VERSION_GENERIC = $(shell pulumictl convert-version --language generic --version "$(PROVIDER_VERSION)")
+
+# Use this normalized version everywhere rather than the raw input to ensure consistency.
+VERSION_GENERIC ?= $(shell make VERSION_GENERIC=$(PROVIDER_VERSION) version_generic) # recur to auto-install pulumictl
```

Comment:
Without the recursive call to make, it seems a bigger refactor is required as `VERSION_GENERIC` expansion is run very early in the cycle by Make and is not preceded by any target execution. The recursive call avoids the larger refactor. Note we need to seed the VERSION_GENERIC variable with an incorrect value to avoid an infinite recursion.